cli/proc_state.rs

```diff
@@ -695,3 +696,21 @@ fn source_map_from_code(code: String) -> Option<Vec<u8>> {
     None
   }
 }
+
+fn binary_env_var(name: &str) -> bool {
+  if let Ok(v) = env::var(name) {
+    if v == "1" {
```

Comment:
You know my position on this, we should accept "true" and "false" for consistency.

This also works with `NO_COLOR` ([evidence](https://github.com/denoland/deno/blob/main/cli/tests/integration/test_tests.rs#L12)).